docs/releases/4.1.txt

```diff
@@ -221,6 +221,10 @@ Management Commands
 * The new :option:`migrate --prune` option allows deleting nonexistent
   migrations from the ``django_migrations`` table.
 
+* Python files created by :djadmin:`startproject`, :djadmin:`startapp`,
+  :djadmin:`makemigrations`, and :djadmin:`squashmigrations` will be formatted
```

Comment:
```suggestion
  :djadmin:`makemigrations`, and :djadmin:`squashmigrations` are now formatted
```

Reply:
Maybe we should add a small section to the [Extra niceties](https://docs.djangoproject.com/en/4.0/ref/django-admin/#extra-niceties) :thinking: